src/main/java/com/amazon/asksdk/cloudfoody/OpenTable.java

```diff
@@ -0,0 +1,42 @@
+package com.amazon.asksdk.cloudfoody;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.*;
```

Comment:
NP: It is the best practice to avoid import all (.*) syntax and import individual items.

Reply:
Done